core/src/main/java/com/linecorp/armeria/server/ServerMetrics.java

```diff
@@ -174,6 +190,10 @@ public void bindTo(MeterRegistry meterRegistry) {
         meterRegistry.gauge(allRequestsMeterName,
                             ImmutableList.of(Tag.of("protocol", "http1.websocket"), Tag.of("state", "active")),
                             activeHttp1WebSocketRequests);
+        // pending non-transient responses
+        meterRegistry.gauge(allRequestsMeterName,
```

Comment:
Question) Can we keep the old format (`armeria.server.pending.responses`) so that metric name related breaking changes aren't made?